vendor/init.bat

```diff
@@ -317,6 +320,19 @@ echo.
 echo :: you can add your plugins to the cmder path like so
 echo :: set "PATH=%%CMDER_ROOT%%\vendor\whatever;%%PATH%%"
 echo.
+echo :: Pass custom arguments to init.bat, and you can parse them like so:
+echo :: If found...
+echo :: echo %* | find /i "/noautorun">nul
+echo :: if "%ERRORLEVEL%" == "0" (
+echo ::   call vsCode
```

Comment:
is `vsCode` a batch file, or a subroutine? Why is it called here?

Reply:
It's not it's a comment